piccolo/table.py

```diff
@@ -407,6 +416,52 @@ def get_related(self, foreign_key: t.Union[ForeignKey, str]) -> Objects:
             .first()
         )
 
+    # TODO - I might merge this with ``get_related``.
+    def get_m2m(self, m2m: M2M) -> M2MGetRelated:
+        """
+        Get all matching rows via the join table.
+
+        .. code-block:: python
+
+            >>> band = await Band.objects().get(name="Pythonistas")
+            >>> await band.get_m2m(Band.genres)
+            [<Genre: 1>, <Genre: 2>]
+
+        """
+        return M2MGetRelated(row=self, m2m=m2m)
+
+    def add_m2m(
+        self,
+        *rows: Table,
+        m2m: M2M,
+        extra_column_values: t.Dict[t.Union[Column, str], t.Any] = {},
+    ) -> M2MAddRelated:
+        """
+        Save the row if it doesn't already exist in the database, and insert
+        an entry into the joining table.
+
+        .. code-block:: python
+
+            >>> band = await Band.objects().get(name="Pythonistas")
+            >>> await band.add_m2m(
+            >>>     Genre(name="Punk rock"),
+            >>>     m2m=Band.genres
+            >>> )
+            <Genre: 5>
```

Comment:
Same here. Minor typo here it should be `await Band.objects().get(Band.name == "Pythonistas")` and for `line 450` I get result like this `[{'id': 5}]`

Reply:
Thanks, fixed.